components/pagination/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@hods/pagination",
+  "version": "0.1.8",
+  "description": "Component to allow separating long tables into separate tables",
+  "main": "src/Pagination.tsx",
+  "sass": "assets/Pagination.scss",
+  "publishConfig": {
+    "main": "dist/Pagination.js",
+    "typings": "dist/Pagination.d.ts"
+  },
+  "files": [
+    "/assets",
+    "/dist"
+  ],
+  "scripts": {
+    "test": "jest",
+    "prepublishOnly": "npm run clean && npm run build",
+    "build": "tsc",
+    "clean": "rm -rf dist tsconfig.tsbuildinfo"
+  },
+  "author": "Daniel A.C. Martin <npm@daniel-martin.co.uk> (http://daniel-martin.co.uk/)",
+  "license": "MIT",
+  "keywords": [
+    "react-components"
+  ],
+  "dependencies": {
+    "@not-govuk/anchor": "^0.2.9",
+    "@not-govuk/component-helpers": "^0.2.9",
+    "@not-govuk/docs-components": "^0.2.9",
+    "@not-govuk/sass-base": "^0.2.9",
+    "react": "^17.0.2"
```

Comment:
You mustn't do this.

React is a peer dependency. Setting is as a normal dependency will break things.

Reply:
this must've been done by a command - possibly `npm i` or similar. I haven't touched any package file. Have amended this in the latest commit